src/components/vm/hostdevs/hostDevAdd.scss

```diff
@@ -1,4 +1,4 @@
-@import "_global-variables.scss";
+@use "_global-variables.scss";
```

Comment:
- Switching to use breaks the point of the import.
- However, I don't see anything using variables in this file, so the line itself is probably incorrect and should probably be removed anyway? 
- Also: Imports should only exist at toplevel in most cases, as these sub-files that are also included in a page will pull in duplicate CSS. (Of course, anything that relies on SASS variables (not CSS custom properties) should be imported. This is the exception. And anything that needs to pull in SASS variables should not have CSS in the file that it is importing.)

Reply:
This was originally introduced in `818cb5d10b718c3933246c93dde64617c701a3b6` I think for `--pf-c-table--m-compact-th--PaddingTop`? But no, it compiles with out it.

So I guess `--pf-c-table--m-compact-th--PaddingTop` is just implicitly available through one of our existing imports, in that case I hope we have a pixel test for that dialog.